### PR TITLE
kdialog: add page

### PR DESCRIPTION
**Pull request by marchersimon**
**Version of the command being documented (if known):** kdialog 22.04.0

Finally back in business

### CODE REVIEW

File: pages/linux/kdialog.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# kdialog
+
+> Show KDE dialog boxes from within shell scripts.
+> More information: <https://develop.kde.org/deploy/kdialog/>.
+
+- Open a dialog box displaying a custom message:
```
Comment (EmilySeville7cfg) [inline pages/linux/kdialog.md:6]:
```suggestion
- Open a dialog box displaying a specific message:
```

File: pages/linux/kdialog.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Open a warning dialog with a `yes`, `no`, and `cancel` button, returning `0`, `1`, or `2` respectively:
+
+`kdialog --warningyesnocancel "{{message}}"`
+
+- Open an input dialog box and print the input to stdout when `OK` is pressed:
+
+`kdialog --inputbox "{{message}}" "{{optional_default_text}}"`
+
+- Open a dialog to prompt for a password and print it to stdout:
```
Comment (EmilySeville7cfg) [inline pages/linux/kdialog.md:22]:
```suggestion
- Open a dialog to prompt for a specific password and print it to stdout:
```

File: pages/linux/kdialog.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`kdialog --password "{{message}}"`
+
+- Open a dialog containing a dropdown menu and print the selected item to stdout:
```
Comment (EmilySeville7cfg) [inline pages/linux/kdialog.md:26]:
```suggestion
- Open a dialog containing a specific dropdown menu and print the selected item to stdout:
```

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/linux/kdialog.md (hunk 2 above)
Comment (marchersimon) [reply pages/linux/kdialog.md:22]: I'm not so sure about this one. It isn't really a "specific password", it's just "any password".

File: pages/linux/kdialog.md (hunk 3 above)
Comment (marchersimon) [reply pages/linux/kdialog.md:26]: How about `predefined` instead?

File: pages/linux/kdialog.md (hunk 2 above)
Comment (navarroaxel) [reply pages/linux/kdialog.md:22]: It's like GPG, the prompt ask for a password for a specific service, or `sudo`. So it asks for a specific password.

File: pages/linux/kdialog.md (hunk 3 above)
Comment (EmilySeville7cfg) [reply pages/linux/kdialog.md:26]: Although it's a good idea but to make all pages more standardized it's better to stick to smth one. 😄 For instance to `specific`.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**